### ISSUE TITLE
bundle and bundle-sfx breaks CSS rules

### ISSUE THREAD

**Issue by blittle** [geelen/jspm-loader-css-modules#5]
@geelen great job speaking at React Rally. We have been trying CSS Modules out in within an internal application. It seems as though building a bundle (sfx or normal) breaks css rules. The output bundle correctly provides a JS module with the appropriately generated class names that are loaded into the JavaScript. It does not include any css declarations with the generated class names.

I have a minimal example repository that shows the problem:

https://github.com/blittle/jspm-cssmodule-test

If you clone the repository and run the following:
- `npm run all` - Runs npm install, jspm install, jspm bundle-sfx, and starts a static server on port 1337
- Navigate to [http://localhost:1337/](http://localhost:1337/) and notice the background style is applied.
- Navigate to [http://localhost:1337/index.build.html](http://localhost:5555/index.build.html) and notice the background style is **not** applied.

The only difference between the two is the latter is running a `jspm bundle-sfx` script.

The app.js file is:

``` javascript
import React from 'react';
import ReactDOM from 'react-dom';

import styles from './app.css!';

function App() {
  return (
    <div className={styles.myRule}>Hello</div>
  )
}
```

The app.css file is:

``` css
.myRule {
  background: red;
}
```

The output sfx bundle properly includes the generated className: 

``` javascript
$__System.register("a4", [], function(_export) {
  "use strict";

  return {
    setters: [],
    execute: function() {
      _export("default", {
        "myRule": "_app__myRule"
      });
    }
  };
});
```

But there is no instance of the generated rule definition `_app__myRule`

**Comment by blittle** [comment]
This does not appear to be a problem in https://github.com/geelen/jspm-loader-css so that is my current work around (though I'd prefer on new projects to always opt in to CSS modules with this loader)

**Comment by vhiroki** [comment]
+1

**Comment by abubics** [comment]
Ja, dealbreaker for me :'(

**Comment by juliogarciag** [comment]
@geelen I think that was done in the forks noted above (exposing `bundle`) is the right solution. Do you mind to implement that and release a version of this? Right now, this is preventing people from using this (awesome) package in production and the solution seems pretty simple. It would be better if a forker (@oligot or @jhoch) creates a PR and you only accept it and create a release. Well, anyway, it wouldn't take too much time i guess.

Greetings,

**Comment by oligot** [comment]
There is already a PR here: #8

**Comment by jhoch** [comment]
@juliogarciag I agree, but I saw the PR was stale and wanted to play around with things in the meantime.  I'll close my fork as soon as @oligot's PR merges - thanks for the fix!
